### DISCUSSION TITLE
PlusToolkit Ascension and Ultrasound Configuration

### DISCUSSION THREAD

**Discussion by rachelbonfini** [PlusToolkit/PlusLib#922] [Q&A]
Hello! I am very new to plus toolkit and am hoping someone can help me out with the configuration code I need. I am trying to include Ascension 3DG as my tracker device and use Siemens' Acuson P300 Ultrasound as another device. Would there be an existing configuration that could do this or would anyone know how to form a configuration with these devices? Thank you in advance.

**Comment by ungi** [comment]
Hi! Ascension tracking should be no problem. We have lots of examples for that. The question is how you will get images from the ultrasound machine. If you need images in real time and you find a video output on the ultrasound machine somewhere, then you could use a frame grabber supported by PLUS. That would capture the entire screen of the ultrasound machine. If there is no video output connector on the machine, then you may talk to Siemens and ask if they have any kind of real time research interface. Most likely that is not something they will offer. If none of these work, I would consider switching to another ultrasound machine that either has a research interface or at least a video output.

**Reply by rachelbonfini** [reply]
Thank you for the information! Sorry about the late response. I have the Siemens ultrasound connected to ClonerBox to record the ultrasound screen. Is there a way to connect this to PLUS so that I can see the realtime video on 3D slicer?
